sdk/core/Azure.Core/tests/OperationInternalOfTTests.cs

```diff
@@ -518,6 +518,8 @@ public TestOperation(Func<bool, CancellationToken, ValueTask<OperationState<int>
             }
 
             public ValueTask<OperationState<int>> UpdateStateAsync(bool async, CancellationToken cancellationToken) => _updateStateAsyncHandler(async, cancellationToken);
+
+            public RehydrationToken GetRehydrationToken() => default;
```

Comment:
should these throw like the actual implementations?

Reply:
We could. These are only part of a test project and also never invoked, so it doesn't matter much.

Reply:
The test should validate that it actually never does get called 😄

Reply:
That's reasonable 👀 I'll throw `NotImplemented` from these.

Reply:
Updated.